episodes/08-connected-components.md

```diff
@@ -316,24 +316,22 @@ labeled_image, count = connected_components(filename="data/shapes-01.jpg", sigma
 
 fig, ax = plt.subplots()
 plt.imshow(labeled_image)
-plt.axis("off");
+plt.axis("off")
```

Comment:
I would keep the semicolon, so the output is not cluttered with text and memory addresses... But maybe the semicolon should be explained to learners? cf. https://github.com/mkcor/python-prog/pull/8#discussion_r1158597006 :wink:

Reply:
Sure, sorry, I probably shouldn't have removed this! I don't use Jupyter much apart from for teaching so from my perspective I thought it might be confusing for participants and I didn't mind seeing the limits of the axis printed. Happy to put back in with a brief explanation :)